pygdf/numerical.py

```diff
@@ -141,17 +141,18 @@ def _unique_segments(self):
         segs, begins = cudautils.find_segments(sortedvals)
         return segs, sortedvals
 
-    def unique(self):
+    def unique(self, type='hash'):
```

Comment:
We should add the templating here for checking the type and if someone passes "hash" it raises a not implemented exception, and the default should be "sort" for now.